src/index.ts

```diff
@@ -1,6 +1,8 @@
 import { config } from './config'
 import { CameraController } from './cam'
 
+type ControlsKeys = 'ArrowLeft' | 'ArrowRight' | 'ArrowUp' | 'ArrowDown' | '=' | '-' | '/' | 'o'
```

Comment:
Instead of creating a new type for ControlsKeys, maybe we should use the already defined keys inside controls variables, like:
```ts
const controls = {
  ArrowLeft: () => cameraController.adjustOffset('left'),
  ArrowRight: () => cameraController.adjustOffset('right'),
  ArrowUp: () => cameraController.adjustOffset('up'),
  ArrowDown: () => cameraController.adjustOffset('down'),
  '=': () => cameraController.zoom('in'),
  '-': () => cameraController.zoom('out'),
  '/': () => cameraController.flipHorizontal(),
  o: () => cameraController.round(),
}

type ControlsKeys = keyof typeof controls;
```
It'll be a little bit easier to add new control keys as we don't need to maintain the type separately. 

What u think about?